java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -74,6 +76,45 @@ public interface StringBaseCommands {
      */
     CompletableFuture<String> getdel(String key);
 
+    /**
+     * Gets the value associated with the given <code>key</code>. <code>GETEX</code>is similar to
+     * {@see #get}.
+     *
+     * @since Redis 6.0.6.
```

Comment:
Same for other docs
```suggestion
     * @since Redis 6.2.0.
```

Reply:
redis [documentation](https://redis.io/docs/latest/commands/getex/) for it